docs/src/rules/no-loop-func.md

```diff
@@ -70,6 +70,25 @@ for (let i = 0; i < 10; ++i) {
     setTimeout(() => console.log(foo));
 }
 foo = 100;
+
+var arr = [];
+
+for (var i = 0; i < 5; i++) {
+    arr.push((f => f)(() => i));
+}
+
+for (var i = 0; i < 5; i++) {
+    arr.push((() => {
+        return () => i;
+    })());
+}
+
+for (var i = 0; i < 5; i++) {
+    (function fun () {
+        if (arr.includes(fun)) return i;
+        else arr.push(fun);
+    })();
+}
```

Comment:
Is there a reason you used `var` in these examples instead of `let` as in previous ones?

Reply:
Yeah, it works as expected with `let`.

with `var` it would return [5, 5, 5, 5, 5] with let: `[0, 1, 2, 3, 4]`